### PR TITLE
Readds Birdboatstation

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/40907
----------------------------
:cl: Bawhoppen & Tokiko
add: Birdboatstation is back.
/:cl:

After numerous hours of updating paths and manual fixes, Birdboatstation (the best map ever made) is back. 
Birdboat is a lowpop map, originally made by Tokiko, which boasts a unique paranoia-laden design (TM) and several job modifications. 
The new version includes various updates and modernizations, such as techfabs and a working arrivals shuttle, among other things. It's likely missing some different newer features, but those can be added in as they're discovered.

FastDMM Map Image: https://www.dropbox.com/s/3sd4ly4zeskpcgm/Birdboat2Image3.png?dl=0
Old 2015 picture for reference: https://tgstation13.org/wiki//images/9/92/Birdstation1.png

### CODE REVIEW

**Comment by k-k-karma** [comment]
requested port from a DNM tg pr, treat like a normal PR please

File: README.md [hunk 1]
```diff
@@ -100,12 +100,13 @@ recompile the game.  Once you start the server up again, you should be running
 the new version.
 
 ## MAPS
-Hippie Station currently comes equipped with five maps.
+/tg/station currently comes equipped with five maps.
```
Comment (k-k-karma) [inline README.md]:
```suggestion
Hippie Station currently comes equipped with five maps.
```

**Comment by yskefunky** [comment]
birdboat is great and all but wheres my og nigga donut 
+1 anyway

**Comment by yobob591** [comment]
I will always +1 more maps, though I'd prefer if all of them had the pool and other related hippie exclusive places

**Pull request closed without merging**